csrc/ir/utils.h

```diff
@@ -715,4 +715,12 @@ bool hasRootToLoopLinearTransformations(const TensorView* tv);
 //! also checks the loop domain has any extra domain
 bool isLoopDomainFullyDerivedFromLogicalDomain(TensorView* tv);
 
+//! If the given statement is nullptr, return "nullptr", otherwise return its
```

Comment:
I personally encountered this use case multiple times, so I added it as a utility so that I can use it for the future. I don't remember where I used it, otherwise I would have refactored these points to use this new utility.